src/rollup/types.d.ts

```diff
@@ -89,8 +89,8 @@ export interface PluginCache {
 }
 
 export interface PluginContext {
-	// TODO deprecate:
-	watcher: Watcher;
+	/** @deprecated */
```

Comment:
👍 Very good, works nicely in my IDE:
<img width="502" alt="bildschirmfoto 2018-08-15 um 09 04 38" src="https://user-images.githubusercontent.com/12949268/44135677-6638f31e-a06a-11e8-81a2-2544b4bddb83.png">